internal/component/prometheus/exporter/cloudwatch/config.go

```diff
@@ -310,19 +343,21 @@ func toYACEDiscoveryJob(rj DiscoveryJob) *yaceConf.Job {
 		CustomTags:                rj.CustomTags.toYACE(),
 		SearchTags:                rj.SearchTags.toYACE(),
 		DimensionNameRequirements: rj.DimensionNameRequirements,
-		// By setting RoundingPeriod to nil, the exporter will align the start and end times for retrieving CloudWatch
-		// metrics, with the smallest period in the retrieved batch.
-		RoundingPeriod:     nil,
```

Comment:
thought: removed RoundingPeriod as an option as this IS deprecated according to YACE: https://github.com/prometheus-community/yet-another-cloudwatch-exporter/blob/0c9677d91836f0a4150a55172a0ce5081574b407/pkg/config/config.go#L258